[2021-02-23 14:56:26,416] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: tutorial.print_date 2021-02-21T00:00:00+00:00 [queued]>
[2021-02-23 14:56:28,169] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: tutorial.print_date 2021-02-21T00:00:00+00:00 [queued]>
[2021-02-23 14:56:28,254] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:56:28,262] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-02-23 14:56:28,268] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:56:28,773] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): print_date> on 2021-02-21T00:00:00+00:00
[2021-02-23 14:56:29,301] {standard_task_runner.py:52} INFO - Started process 57 to run task
[2021-02-23 14:56:31,296] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'tutorial', 'print_date', '2021-02-21T00:00:00+00:00', '--job-id', '2', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/tutorial.py', '--cfg-path', '/tmp/tmpay84oidy', '--error-file', '/tmp/tmpmer3cx7p']
[2021-02-23 14:56:31,324] {standard_task_runner.py:77} INFO - Job 2: Subtask print_date
[2021-02-23 14:56:34,019] {logging_mixin.py:104} INFO - Running <TaskInstance: tutorial.print_date 2021-02-21T00:00:00+00:00 [running]> on host 0513e3a4eab6
[2021-02-23 14:56:36,224] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2021-02-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-21T00:00:00+00:00
[2021-02-23 14:56:36,326] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-02-23 14:56:36,439] {bash.py:158} INFO - Running command: date
[2021-02-23 14:56:37,564] {bash.py:169} INFO - Output:
[2021-02-23 14:56:38,096] {bash.py:173} INFO - Tue Feb 23 14:56:38 UTC 2021
[2021-02-23 14:56:38,460] {bash.py:177} INFO - Command exited with return code 0
[2021-02-23 14:56:42,860] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=print_date, execution_date=20210221T000000, start_date=20210223T145626, end_date=20210223T145642
[2021-02-23 14:56:45,583] {taskinstance.py:1220} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-02-23 14:56:45,809] {local_task_job.py:188} WARNING - State of this instance has been externally set to success. Terminating instance.
[2021-02-23 14:56:45,919] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 57
[2021-02-23 14:56:46,004] {process_utils.py:66} INFO - Process psutil.Process(pid=57, status='terminated', exitcode=0, started='14:56:28') (57) terminated with exit code 0
